autotest/gcore/misc.py

```diff
@@ -158,7 +158,7 @@ def misc_5_internal(drv, datatype, nBands):
         if drv.ShortName not in ['PNM', 'MFF', 'NULL']:
             got_gt = ds.GetGeoTransform()
             for i in range(6):
-                if abs(got_gt[i] - set_gt[i]) > 1e-10:
+                if got_gt[i] != pytest.approx(set_gt[i], abs=1e-10):
                     print('Did not get expected GT for drv = %s, nBands = %d, datatype = %s' % (drv.ShortName, nBands, gdal.GetDataTypeName(datatype)))
                     print(got_gt)
                     return -1
```

Comment:
OT: thought these were all gone now?

Reply:
Most are. Guess I missed a few due to them not fitting the usual pattern.